src/app/components/main/header/header.component.html

```diff
@@ -1,6 +1,4 @@
-<!--
- Header - navbar
- -->
+<!--Todo - Add an entry to the `My events` dropdown to be able to navigate to the overview 2. -->
```

Comment:
Technically you could remove this, because you've already done below what's asked.

Reply:
We'll get into routing at a later stage in the project, don't worry about that for now.

Reply:
Alrighty